src/quo2/core.cljs

```diff
@@ -301,3 +303,7 @@
 (def network-amount quo2.components.wallet.network-amount.view/view)
 (def network-bridge quo2.components.wallet.network-bridge.view/view)
 (def account-card quo2.components.wallet.account-card.view/view)
+
+;;;; SWITCHER
+(def switcher-base-card quo2.components.switchers.base-card.view/base-card)
```

Comment:
`base-card` is internal to the `quo2` module. we should only export 1-1 with Figma from the quo2.core, as such this does not need a preview screen either 👍